dist/wearable/js/tau.js

```diff
@@ -30404,6 +30404,20 @@ function pathToRegexp (path, keys, options) {
 				element.parentElement.insertBefore(buttonsContainer, element);
 			};
 
+			/**
```

Comment:
This change should be applied to `/src/js/profile/wearable/widget/wearable/Slider.js`.
The `dist` folder is overriden with each build.

I agree with @pkaczmarczy about store the `buttonsContainer` in `this._ui` object.